kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -523,8 +523,9 @@ def exec_foundry_view_kcfg(foundry_root: Path, test: str, **kwargs: Any) -> None
     foundry = Foundry(foundry_root)
     ag_proofs_dir = foundry.out / 'ag_proofs'
     contract_name = test.split('.')[0]
+    proof_digest = f'{test}:{foundry.contracts[contract_name].digest}'
```

Comment:
Consider making this a method on `Foundry`.

Reply:
Even better, you can hide all these details in `Foundry` and expose `ag_proof(test: str)`.

Reply:
I've factored out `Foundry.proof_digest` for now, since that was easy and a pre-req. Factoring out `Foundry.proof` is harder because of how the `setUp` functions work, but I've made an issue: https://github.com/runtimeverification/evm-semantics/issues/1769